virtcontainers/api.go

```diff
@@ -590,6 +590,10 @@ func statusContainer(sandbox *Sandbox, containerID string) (ContainerStatus, err
 				}
 
 				if !running {
+					virtLog.WithFields(logrus.Fields{
+						"state":       container.state.State,
+						"process pid": container.process.Pid}).
```

Comment:
Nit: By (admittedly undocumented) convention we try to ensure all log field names do not contain spaces, so could you change this to `process-id` (or even just `pid` as I think that's a well-understood term).

Reply:
I'll use the `process-pid` since we already have a `pid` field in `virtLog`, so if I use `pid` here that will confuse developers.